ansible/group_vars/sonic/variables

```diff
@@ -11,6 +11,7 @@ broadcom_td3_hwskus: ['Arista-7050CX3-32S-C32', 'Arista-7050CX3-32S-D48C8']
 broadcom_th_hwskus: ['Force10-S6100', 'Arista-7060CX-32S-C32', 'Arista-7060CX-32S-C32-T1', 'Arista-7060CX-32S-D48C8', 'Celestica-DX010-C32', "Seastone-DX010" ]
 broadcom_th2_hwskus: ['Arista-7260CX3-D108C8',  'Arista-7260CX3-C64', 'Arista-7260CX3-Q64']
 broadcom_th3_hwskus: ['DellEMC-Z9332f-M-O16C64',  'DellEMC-Z9332f-O32']
+broadcom_j2c+_hwskus: ['Nokia-IXR7250E-36x100G', 'Nokia-IXR7250E-36x400G']
 
```

Comment:
![image](https://user-images.githubusercontent.com/46945843/189765358-fbb7f7dd-3bc1-4b99-bba8-2ee7bab4e047.png)
Can you please add jr2 hwsku as well ?

Reply:
@vmittal-msft - please add Arista linecard

Reply:
Pushed commit to add this